api/src/main/resources/emails/tier_access_expired/content.html

```diff
@@ -1,3 +1,4 @@
+<!-- FILE: content.mjml -->
```

Comment:
My version of mjml added this

Reply:
That's strange. Did you install mjml globally? if so, which version does it say you have in ```vim /usr/local/lib/node_modules/mjml/package.json```

Reply:
-o removes it, so now my version matches.  Updated.